content/en/real_user_monitoring/browser/setup/server/ibm.md

```diff
@@ -117,4 +117,14 @@ If you notice that RUM is not being injected into HTML pages, consider the follo
 [2]: /agent/
 [3]: https://httpd.apache.org/modules/
 [4]: https://rum-auto-instrumentation.s3.amazonaws.com/httpd/latest/mod_datadog-amd64.zip
-[5]: https://app.datadoghq.com/rum/list
\ No newline at end of file
+[5]: https://app.datadoghq.com/rum/list
+
+## Uninstall
+
+While automatic uninstallation will be available soon, you can manually remove the RUM Auto-Instrumentation by following these steps.
+
+1. Locate your IBM HTTP Server binary, it may be one of `httpd`, `apachectl`, `apache2` or `apache2ctl`. `httpd` will be used for illustration purposes.
```

Comment:
```suggestion
1. Locate your IBM HTTP Server binary, it may be one of `httpd`, `apachectl`, `apache2` or `apache2ctl`. The following steps use `httpd` as an example.
```